### PR TITLE
Add new page for Schematron files and validation reports

### PR DESCRIPTION
**Pull request by bertrand-lorentz**
This targets only 1.6.x, as the new page describes information added in that SDK version.

### CODE REVIEW

File: modules/schematrons/pages/index.adoc [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+`location`:: The exact location that was matched by the rule context, as an absolute XPath.
+`test`:: The XPath expression of the check that failed.
+`role`:: The severity of the failure, either `ERROR` or `WARN`.
+`flag`:: A specific characteristic of the failed assertion. The value `LAWFULNESS` indicates that the failure indicates the notice might not be suitable to be published.
+
+When relevant, additional information is provided via the `diagnostic-reference` element:
+
+[horizontal]
+`diagnostic`:: This attribute contains the identifier of the diagnostic information. This should not be used to extract information from the report.
+`see`:: This attribute contains the identifier of the node or field that is targetted by the failed assertion. The value starts either with `node:` or `field:` to distinguish the 2 types of identifiers.
```
Comment (rouschr) [inline modules/schematrons/pages/index.adoc:91]: targetted should be targeted

**Review by rouschr — APPROVED** [review]
Just see my comment about "targetted", but OK for me.

**Review by pdonohoe — APPROVED** [review]
Removed personal pronouns
Updated paragraph describing CVS translation of message IDs
Fixed bug in procedure-lot-part-information.adoc

File: modules/schematrons/pages/index.adoc (hunk 1 above)
Comment (bertrand-lorentz) [reply modules/schematrons/pages/index.adoc]: This has been corrected in Paul's commit.

**Pull request merged**